Src/IronPython/Modules/Builtin.cs

```diff
@@ -1511,6 +1516,12 @@ [NotNull]IList enumerator
             }
         }
 
+        /// <summary>
+        /// Calls the __next__ method of an iterable
+        /// </summary>
+        /// <param name="iter">Iterable instance</param>
+        /// <param name="defaultVal">Default operation value</param>
+        /// <returns>Next object or throws an StopIterator exception</returns>
         public static object next(IEnumerator iter, object defaultVal) {
```

Comment:
No, because this is the built-in  `next` method, not the one that must be implemented:
```python
...     def __next__(self):
...         return next(self.l)
```